projects/ccd-case-ui-toolkit/src/lib/shared/components/case-viewer/case-specific-access-request/case-specific-access-request.component.html

```diff
@@ -5,6 +5,10 @@
   {{'Authorisation is needed to access this case.' | rpxTranslate}}<br />
   {{'This could be because it\'s outside your jurisdiction, or you may be excluded from the case. If you request access to this case, it will be logged for auditing purposes.' | rpxTranslate}}
 </cut-alert>
+<div *ngIf="getSpecificAccessError" class="govuk-grid-column-two-thirds">
```

Comment:
Can we not use the <exui-error-message> component at the top, rather than adding a new element? You would need to change the ngIf to 
 *ngIf="(formGroup.invalid && submitted) || getSpecificAccessError"

Reply:
Actually I was thinking of to do use the same component but was getting issue in unit test where one of the the test keep on failing, but now it has been rectified and updated as mentioned above